adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +319,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
+        return mIsExtendedLifeTimeToken;
+    }
+
+    /**
+     * Sets the flag to indicate whether the token being returned is a token only
+     * valid in terms of extended lifetime
+     *
+     * @param isExtendedLifeTimeToken
+     */
+    final void setIsExtendedLifeTimeToken(final boolean isExtendedLifeTimeToken) {
+        this.mIsExtendedLifeTimeToken = isExtendedLifeTimeToken;
+    }
+
+    final void setExtendedExpiresOn(final Date extendedExpiresOn) {
+        this.mExtendedExpiresOn = extendedExpiresOn;
```

Comment:
>this [](start = 8, length = 4)

we should not do this. makes no sense unless the local variable and class variable have the same name.